iphone/Classes/TiApp.m

```diff
@@ -100,12 +100,19 @@ -(TiContextGroupRef)contextGroup
 	return contextGroup;
 }
 
-
 +(TiContextGroupRef)contextGroup
 {
 	return [sharedApp contextGroup];
 }
 
+- (NSMutableDictionary *)queuedBootEvents
+{
+    if (queuedBootEvents == nil) {
+        queuedBootEvents = [[[NSMutableDictionary alloc] init] retain];
```

Comment:
Retaining it to ensure it doesn't get GC'd during other lifetime events. I noticed that this sometimes happens and sometimes not, so I ensure to retain it here. Can you run the analyser to see if it's fine with it? Thanks!